frontend/ui/elements/reader_menu_order.lua

```diff
@@ -121,6 +121,7 @@ local order = {
         "news_downloader",
         "send2ebook",
         "text_editor",
+        "moveToArchive",
```

Comment:
Is more plugins always on the next page? I meant the menu item just under this one.

```suggestion
        "move_to_archive",
```

Reply:
done. 

> Is more plugins always on the next page? I meant the menu item just under this one.

I think it depends on your screen resolution + no. of enabled plugins.

Any I made it last one as you wanted...